### PR TITLE
MODORGS-14 - Schema Updates

### PR DESCRIPTION
**Pull request by khandramai**
[MODORGS-14](https://issues.folio.org/browse/MODORGS-14) - Schema Updates

## Purpose
Schemas updating

## Approach
Schemas, examples, reference data updated according to the requirements:

- Fields should use camelCase instead of underscore_notation
- interface.delivery_method should be an enumeration: Online, FTP, Email, Other
- account.payment_method should be an enumeration: Cash, Credit Card, EFT, Deposit Account, Physical Check, Bank Draft, Internal Transfer, Other
- Remove first and last fields from *collection schemas

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Examples exist for all schemas
  - [x] Descriptions exist for all schema properties
  - [x] All schemas pass raml-cop linting
- Does this introduce breaking changes?
  - [ ] Were there any schema changes?
  - [x] There are no breaking changes in this PR.

### CODE REVIEW

File: mod-orgs/schemas/account.json [hunk 1]
```diff
@@ -13,35 +13,45 @@
       "description": "The unique name of this account",
```
Comment (piotr-kalashuk) [inline mod-orgs/schemas/account.json]: Could you please fix doubled `description` for `id` property above?

File: mod-orgs/schemas/agreement.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
       "description": "The discount amount of this agreement",
```
Comment (piotr-kalashuk) [inline mod-orgs/schemas/agreement.json]: Could you please fix doubled `description` for `id` property above?

File: mod-orgs/schemas/account.json (hunk 1 above)
Comment (pautau-s) [reply mod-orgs/schemas/account.json]: Do we even need an id after we removed the API for accounts?

File: mod-orgs/schemas/agreement.json (hunk 2 above)
Comment (pautau-s) [reply mod-orgs/schemas/agreement.json]: Do we even need an id after we removed the API for agreements?

File: mod-orgs/schemas/alias.json [hunk 3]
```diff
@@ -1,12 +1,11 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
-  "description": "A organization alias",
+  "description": "An organization alias",
   "type": "object",
   "properties": {
     "id": {
```
Comment (pautau-s) [inline mod-orgs/schemas/alias.json]: Do we even need an id after we removed the API for aliases?

File: mod-orgs/examples/contact_collection.sample [hunk 4]
```diff
@@ -3,14 +3,14 @@
...
+            "lastName": "Fury",
             "language": "en-us",
             "notes": "SHIELD's Big Kahuna",
-            "phone_numbers": [
+            "phoneNumbers": [
                 {
-                    "phone_number": {
-                        "phone_number": "19789999999"
+                    "phoneNumber": {
+                        "phoneNumber": "19789999999"
```
Comment (craigmcnally) [inline mod-orgs/examples/contact_collection.sample]: I think you have one too many phoneNumber fields here...  shouldn't this be:

```
            "phoneNumbers": [
                {	                
                        "phone_number": "19789999999"
```

File: mod-orgs/examples/organization_get.sample [hunk 5]
```diff
@@ -34,9 +34,9 @@
       "language": "en"
     }
   ],
-  "phone_numbers": [
+  "phoneNumbers": [
     {
-      "phone_number": "19789999999",
+      "phoneNumber": "19789999999",
```
Comment (craigmcnally) [inline mod-orgs/examples/organization_get.sample]: regarding my earlier comment... I think this is correct

File: mod-orgs/examples/contact_get.sample [hunk 6]
```diff
@@ -1,14 +1,14 @@
...
+    "firstName": "Nick",
+    "lastName": "Fury",
     "language": "en-us",
     "notes": "SHIELD's Big Kahuna",
-    "phone_numbers": [
+    "phoneNumbers": [
         {
-            "phone_number": {
-                "phone_number": "19789999999"
+            "phoneNumber": {
```
Comment (craigmcnally) [inline mod-orgs/examples/contact_get.sample]: again...

File: mod-orgs/examples/contact_post.sample [hunk 7]
```diff
@@ -1,13 +1,13 @@
...
+    "firstName": "Nick",
+    "lastName": "Fury",
     "language": "en-us",
     "notes": "SHIELD's Big Kahuna",
-    "phone_numbers": [
+    "phoneNumbers": [
         {
-            "phone_number": {
-                "phone_number": "19789999999"
+            "phoneNumber": {
```
Comment (craigmcnally) [inline mod-orgs/examples/contact_post.sample]: again

File: mod-orgs/examples/organization_collection.sample [hunk 8]
```diff
@@ -37,10 +37,10 @@
           "language": "en"
         }
       ],
-      "phone_numbers": [
+      "phoneNumbers": [
         {
-          "phone_number": {
-            "phone_number": "19789999999"
+          "phoneNumber": {
```
Comment (craigmcnally) [inline mod-orgs/examples/organization_collection.sample]: again

File: mod-orgs/schemas/alias.json [hunk 9]
```diff
@@ -1,12 +1,11 @@
...
-      "description": "The unique id of this category",
+      "description": "The unique id of this alias",
```
Comment (craigmcnally) [inline mod-orgs/schemas/alias.json]: good catch

**Pull request merged**